telegram/ext/dispatcher.py

```diff
@@ -162,6 +162,11 @@ def __init__(
                 stacklevel=3,
             )
 
+        if self.workers < 1:
+            self.logger.warning(
+                'You must need to have atleast one worker for `run_async` to work properly.'
```

Comment:
Maybe rather something like "Asynchronous callbacks can not be processed without at least one worker thread"?

Reply:
Yea that's better